src/test/scala/inventory/linking/exports/ILERequests.scala

```diff
@@ -0,0 +1,65 @@
+package inventory.linking.exports
+
+import io.gatling.core.Predef._
+import io.gatling.core.check.CheckBuilder
+import io.gatling.http.Predef._
+import io.gatling.http.check.HttpCheck
+import io.gatling.http.request.builder.HttpRequestBuilder
+import jodd.lagarto.dom.NodeSelector
+import uk.gov.hmrc.performance.conf.ServicesConfiguration
+
+object ILERequests extends ServicesConfiguration {
+
+  val authBaseUrl = baseUrlFor("auth")
+  val apiBaseUrl = baseUrlFor("inventory-linking-exports")
+
+  val SuccessfulSubmissionStatusCode = 202
+  val Movement = StringBody(ExamplePayloads.validMovement.toString())
+  val Consolidation = StringBody(ExamplePayloads.validConsolidation.toString())
+  val Query = StringBody(ExamplePayloads.validQuery.toString())
+
+  val SubmissionStatus = "SubmissionStatus"
+  val SubmissionId = "SubmissionId"
+
+  private def headers(user: User): Map[String, String] = Map(
+    "Accept" -> s"application/vnd.hmrc.${user.version}+xml",
+    "Content-Type" -> "application/xml; charset=UTF-8",
+    "Authorization" -> s"Bearer ${user.token}",
+    "X-Client-ID" -> "6372609a-f550-11e7-8c3f-9a214cf093ae",
+    "api-subscription-fields-id" -> "jHKv4OxSi9jEP6PKWTiCNRcgYuQa",
```

Comment:
I think use of `api-subscription-fields-id` is now deprecated. Pls remove line `"api-subscription-fields-id" -> "jHKv4OxSi9jEP6PKWTiCNRcgYuQa",` . In fact in some recent work it processing of it has been taken out of some services. You will have to make sure when running locally that you have seeded data for clientId  `6372609a-f550-11e7-8c3f-9a214cf093ae`